src/operation-node/operator-node.ts

```diff
@@ -40,7 +37,11 @@ export const OPERATORS = [
   '<->',
 ] as const
 
-export type Operator = ArrayItemType<typeof OPERATORS>
+export const UNARY_OPERATORS = ['exists', 'not exists'] as const
```

Comment:
Same here. Something like `UNARY_FILTER_OPERATORS`? Dunno. But not just operators.

Reply:
None of the database documentations agree on this operator's category (oracle lists it under comparison operators, postgresql lists it under subquery expressions) 🤷, so let's just go with `UNARY_FILTER_OPERATORS`.